File: Backend/src/main/java/com/vidasalud/backend/model/RegistroAlimentacion.java
package com.vidasalud.backend.model;

import jakarta.persistence.*;
import lombok.Data;
import java.time.LocalDate;

@Entity
@Data
public class RegistroAlimentacion {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Long usuarioId;
    private LocalDate fecha;
    private Float calorias;
    private Float proteinas;
    private Float grasas;
}

File: App_VidaSalud_Grupo11-Proyecto/app/src/main/java/com/example/app_vidasalud_grupo11/model/RegistroAlimentacion.kt
package com.example.app_vidasalud_grupo11.model

data class RegistroAlimentacion(
    val id: Long? = null,
    val usuarioId: Long,
    val fecha: String? = null,
    val calorias: Float,
    val proteinas: Float,
    val grasas: Float
)